docs/getting-started/install-endpoint.asciidoc

```diff
@@ -107,6 +107,19 @@ After you have installed the agent, malware prevention is automatically enabled
 [role="screenshot"]
 image::images/install-endpoint/malware-protection.png[]
 
+## Register as antivirus (Windows Only)
+
+If you download the Elastic Agent on Windows 10 or above, you can Elastic Security as your antivirus software. To enable Elastic Security as antivirus: 
+
+1. Go to **Fleet** > **Agents**. A list of hosts with the Elastic Agent appears. 
+2. Select your desired host from the list.
+3. From the host's page, find your named **Endpoint** integration and select it. 
```

Comment:
This section makes it sound like the register as AV option will only apply to a selected host.  When you turn the option on, any Windows 10 Host with that particular Agent Policy will receive the "Register as AV" option.

I would drive the user towards choosing a Policy and informing then that any host using that Agent Policy would receive the Register as AV option

Reply:
Kevin: The intention will be to move these steps into an overall Policy overview once this merged in. I think that's help set the context better.